scanning_scripts/modules/smtp_capabilities/scanner.go

```diff
@@ -26,7 +26,7 @@ type ScanResults struct {
 	// PostTLS is the targets response to EHLO in encrypted (post-tls) state
 	PostTLS string `json:"postTLS,omitempty"`
 
-	// Help is the targets response to the HELP command in encrypte (tls) state
+	// Help is the targets response to the HELP command in encrypted (tls) state
```

Comment:
I think this is correct (it's halfway between `encrypt` and `encrypted`).

Note that `targets` should be `target's`, but this is a spell checker, not a grammar check....